dev_docs/contributing/how_we_use_github.mdx

```diff
@@ -191,7 +191,8 @@ These labels categorize the type of work. For example:
 - `docs`/`documentation`: Indicates improvements or additions to documentation
 - `enhancement`: Indicates new feature or enhancement requests
 - `meta`: Indicates that the issue tracks tasks related to a project
-- `needs_triage`: Indicates that someone from the area team needs to investigate.
+- `triage-needed`: Indicates that someone from the area team needs to investigate.
+- `needs_team`: Indicates that issue is missing area team label. All issues should be assigned to one or more area teams for follow up. This label is assigned automatically and removed automatically once a team label is added.
```

Comment:
might be worth standardizing underscore vs dash, but in this case you have them reversed. It should be "triage_needed" and "needs-team" based on the labels today.

Reply:
I saw that and completely agree with you... but I did not want to take on the wrangling of this change as part of this.